contracts/arbitrum/connectors/spell/main.sol

```diff
@@ -0,0 +1,77 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+/**
+ * @title DSA Spell.
+ * @dev Cast spells on DSA.
+ */
+
+// import files
+import { AccountInterface } from "../../common/interfaces.sol";
+import { Stores } from "../../common/stores.sol";
+import { Events } from "./events.sol";
+
+abstract contract DSASpell is Events, Stores {
+	/**
+	 *@dev Cast spells on DSA.
+	 *@param targetDSA target DSA to cast spells on.
+	 *@param connectors Array of connector names.
+	 *@param datas Array of connector calldatas.
+	 */
+	function castDSA(
```

Comment:
Should we name it `castOnDsa` or `castToDsa,` as a regular cast is built for DSA?
This function tries to do a cast on DSA.
@thrilok209 

```suggestion
	function castDSA(
```

Reply:
okay! We can rename to `castOnDsa `

Reply:
Updated.